bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx75.html

```diff
@@ -0,0 +1,153 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/whatsnew_75" %}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ _('What’s new with Firefox') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_75') }}
+{% endblock %}
+
+{% set _entrypoint = 'mozilla.org-whatsnew75' %}
+{% set _utm_campaign = 'whatsnew75' %}
+
+{% block site_header %}{% endblock %}
+
+{% block content %}
+<main class="content-wrapper mzp-t-firefox mzp-t-dark">
+  <header class="c-page-header">
+    <div class="mzp-l-content c-page-header-inner">
+      {{ high_res_img('protocol/img/logos/firefox/browser/logo-word-hor-white-sm.png', {'alt': 'Firefox', 'width': '216', 'height': '40', 'class': 'c-page-header-logo-fx'}) }}
+      <div class="mzp-c-notification-bar mzp-t-success up-to-date">
+        <p>{{ _('Congrats! You’re using the latest version of Firefox.') }}</p>
+      </div>
+    </div>
+  </header>
+
+  <section class="content-main">
+    <div class="mzp-l-content">
+
+      <div class="mzp-c-emphasis-box show-signed-out">
+        <img class="c-emphasis-box-hero" src="{{ static('img/firefox/whatsnew/whatsnew75/hero-sync.svg') }}" alt="">
+        <h2 class="c-emphasis-box-title">{{ _('Pick up Firefox where you left off') }}</h2>
+        <p>{{ _('Whether you’re using Firefox on your phone or computer, make sure all your important stuff — internet searches, passwords, open tabs — appears where you need it.') }}</p>
+
+        <p class="cta">
+          {# NOTE: The '/connect_another_device/' landing page is new so this is a kludge that duplicates the fxa_cta_link macro. If we do this again we should make a macro/helper for this button. #}
```

Comment:
Note for reviewer: this is a kludge. There's likely a better way but I didn't want to try to refactor the fxa_cta_link macro for what may be a one-off button.

It works for me locally where latest.dev.lcip.org is the endpoint. It doesn't currently work on demo3 because that server is configured to use accounts.firefox.com as the endpoint but the demo3 domain isn't whitelisted to fetch the deviceID and flowID. We may need to wait for stage or even prod to do proper end-to-end testing.

Reply:
We're moving away from the FxA macros because they keep getting more complicated with things like this. We should probably file an issue to create a python helper for this new use case. But for now, this works 👍